docs/maui/native-library-interop/get-started.md

```diff
@@ -93,20 +92,30 @@ On the native side, make updates in _template/macios/native/NewBinding/NewBindin
 
 Back on the .NET side, we are now ready to interop with the native library:
 
-1. Run `dotnet build` from _template/macios/NewBinding.MaciOS.Binding_ to test everything is plugged in correctly and good to go. If successful, you should see the generated C# bindings at _template/macios/native/NewBinding/build/sharpie/ApiDefinitions.cs_.
-1. Update the contents of _template/macios/NewBinding.MaciOS.Binding/ApiDefinition.cs_ by replacing it with the contents of _template/macios/native/NewBinding/build/sharpie/ApiDefinitions.cs_.
+1. Run `dotnet build` from _template/macios/NewBinding.MaciOS.Binding_ to test everything is plugged in correctly and good to go. If successful, you should see the generated C# bindings at _template/macios/native/NewBinding/bin/Release/net8.0-ios/sharpie/NewBinding/ApiDefinitions.cs_.
+1. Update the contents of _template/macios/NewBinding.MaciOS.Binding/ApiDefinition.cs_ by replacing it with the contents of _template/macios/native/NewBinding/bin/Release/net8.0-ios/sharpie/NewBinding/ApiDefinitions.cs_.
 1. Run `dotnet build` from _template/macios/NewBinding.MaciOS.Binding_ again.
 
 #### API Definition: Android
 
 On the native side, make updates in _template/android/native/app/src/main/java/com/example/newbinding/DotnetNewBinding.java_:
 
 1. Add an import statement to import the native library you just added.
-2. Write the API definitions for the native library APIs of interest.
-3. Ensure the Android Studio project builds successfully and you are satisfied with the APIs.
+1. Write the API definitions for the native library APIs of interest.
+1. Ensure the Android Studio project builds successfully and you are satisfied with the APIs.
 
 Back on the .NET side, we are now ready to interop with the native library:
-Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
+1. Run `dotnet build` from _template/android/NewBinding.Android.Binding_ to test everything is plugged in correctly and good to go. (Note: This step will require that you have JDK 17 installed)
+1. Reference the Android binding dependency by adding the following code to _template/sample/MauiSample.csproj_. Replace _{yourNewBindingLibrary.aar}_ with your AndroidLibrary .aar file name. (Note: The gradle/maven dependencies often need to be explicitly referenced, as they are not automatically bundled into your library. The _build.gradle.kts_ file is configured to copy depencies into a bin/outputs/deps folder for you to reference in your .NET sample)
+
+```xml
+<ItemGroup Condition="$(TargetFramework.Contains('android'))">
+    <AndroidLibrary Include="..\android\native\newbinding\build\outputs\deps\{yourNewBindingLibrary.aar}">
```

Comment:
```suggestion
    <AndroidLibrary Include="..\android\native\newbinding\bin\outputs\deps\{yourNewBindingLibrary.aar}">
```
Small update for the latest path here

Reply:
Hmm... for me, it is showing up in the build folder? I tried again from the template directly in the latest version of the repo
<img width="348" alt="image" src="https://github.com/user-attachments/assets/16bee959-6f9f-49f4-8db7-3c129d8504f7">

Reply:
Oh sorry I missed an update on the template, there is a ${buildDir} property that can be used over there which should update this path to something like `bin\Release\net8.0-android\outputs\deps`